### DISCUSSION TITLE
Allow for Downloading Images/Videos using the Name of the Post/Comment

### DISCUSSION THREAD

**Discussion by MetalNeo** [Docile-Alligator/Infinity-For-Reddit#1040] [Ideas]
It would be convenient to have an option to save images/videos using the post/comment text as the base filename as opposed to the default filename which usually doesn't stand out when looking at a list of files with similar names.

It could be a toggle in the settings or it could be a prompt when downloading an image.

**Comment by kshorenicholas** [comment]
Will this feature be coming to fruition any time soon? If I recall correctly, Slide has a similar feature as well. Allowing for file names to match the name of the post will allow for additional context,  especially when backing up files.
